icons/reactquery.svg

```diff
@@ -0,0 +1 @@
-3.283a13.174 13.174 0 0 0-.421-1.873c.078-.012.153-.02.232-.05 1.83-.563 3.128-1.17 3.912-1.863.467-.412.771-.874.881-1.382.117-.537.002-1.072-.303-1.577-.516-.832-1.466-1.491-2.818-2.011a15.825 15.825 0 0 0-1.906-.584c.03-.059.042-.117.056-.18.391-1.762.473-3.11.221-4.092-.15-.584-.426-1.063-.844-1.406-.44-.363-.999-.54-1.625-.54-1.032 0-2.12.44-3.28 1.272-.474.34-.968.748-1.477 1.227a1.075 1.075 0 0 0-.135-.14c-1.442-1.197-2.657-1.939-3.696-2.225a3.508 3.508 0 0 0-1.002-.135zm.125.941h.014v.002c1.479 0 3.363 1.303 5.649 3.912a.326.326 0 0 1 .078.207.328.328 0 0 1-.342.317h-.12a33.891 33.891 0 0 0-2.81.092 37.97 37.97 0 0 0-3.363.41c-.183.026-.361-.085-.397-.256l-.02-.092c-.624-3.048-.187-4.581 1.311-4.592zm9.522.002h.004c.534.001.943.233 1.228.698.745 1.213.478 3.421-.806 6.62a.36.36 0 0 1-.155.169.366.366 0 0 1-.47-.121l-.057-.1c-.48-.793-.98-1.548-1.5-2.26a34.084 34.084 0 0 0-2.074-2.555.307.307 0 0 1 .04-.45l.081-.063c1.584-1.289 2.817-1.935 3.709-1.938zm-6.64 5.41v.002h3.69c.227 0 .433.11.546.293l1.847 3.002a.547.547 0 0 1 0 .584l-1.847 2.998a.638.638 0 0 1-.545.297h-3.692a.64.64 0 0 1-.547-.297l-1.847-2.998a.551.551 0 0 1 0-.586l1.847-3a.64.64 0 0 1 .547-.295zm-2.3.233h.002v.004h.002v.002a.392.392 0 0 1 .176.043.31.31 0 0 1 .123.445l-.062.098a30.318 30.318 0 0 0-2.717 5.312.363.363 0 0 1-.447.194l-.102-.03C1.565 13.621.326 12.481 1.11 11.24c.767-1.213 2.997-2.097 6.688-2.65.019-.003.042-.002.057-.002zm2.758.508a.643.643 0 0 0-.549.295L8.67 11.652a.555.555 0 0 0 0 .584l1.393 2.26a.645.645 0 0 0 .548.299h2.774a.646.646 0 0 0 .549-.299l1.388-2.26a.552.552 0 0 0 0-.584l-1.388-2.261a.64.64 0 0 0-.55-.295h-2.773zm8.317.199v.002h.002a.43.43 0 0 1 .136.015l.104.03c3.264 1.035 4.504 2.175 3.72 3.416-.766 1.213-2.995 2.099-6.685 2.652a.366.366 0 0 1-.234-.043c-.168-.09-.229-.287-.13-.441l.071-.1c.504-.787.959-1.574 1.365-2.353.48-.921.93-1.912 1.352-2.967.049-.12.167-.2.299-.211zm-7.824.607h1.789a.64.64 0 0 1 .546.295l.895 1.453a.55.55 0 0 1 0 .586l-.895 1.455a.64.64 0 0 1-.546.295h-1.79a.64.64 0 0 1-.546-.295l-.897-1.455a.55.55 0 0 1 0-.586l.897-1.453a.64.64 0 0 1 .547-.295zm.476.772a.644.644 0 0 0-.549.297l-.42.68a.55.55 0 0 0 0 .585l.42.684a.642.642 0 0 0 .55.293h.837a.64.64 0 0 0 .547-.293l.42-.684a.55.55 0 0 0 0-.586l-.42-.68a.641.641 0 0 0-.547-.296h-.838zm.418.764c.195 0 .375.096.473.253a.476.476 0 0 1 0 .508.556.556 0 0 1-.473.254.554.554 0 0 1-.473-.254.476.476 0 0 1 0-.508.556.556 0 0 1 .473-.254zm-4.932 2.091h.002v.002h.002v.002h.002a.367.367 0 0 1 .326.162l.063.098c.48.793.98 1.547 1.5 2.26a34.178 34.178 0 0 0 2.076 2.55.31.31 0 0 1-.043.454l-.076.062c-2.54 2.067-4.193 2.481-4.955 1.24-.745-1.214-.472-3.421.812-6.62a.349.349 0 0 1 .153-.165c.049-.019.09-.042.138-.045zm10.735 2.473h.013a.334.334 0 0 1 .327.256l.02.094c.627 3.059.183 4.59-1.329 4.59-1.478 0-3.357-1.305-5.643-3.915a.323.323 0 0 1-.082-.205c.002-.176.16-.318.35-.316h.117c.963.003 1.898-.02 2.807-.094a37.964 37.964 0 0 0 3.363-.406v-.002c.019-.003.042-.002.057-.002z"/></svg>
```

Comment:
This still doesn't look quite right, see the comparison below. It looks like you slightly distorted the dimensions this time. When you resize the SVG, be sure that your graphics editor maintains the aspect ratio of the original. In Inkscape you can do this by hitting the "lock" icon between the Width and the Height input at the top, see below, and than setting the Width to 24 (pixels).

##### Comparison

![comparion](https://user-images.githubusercontent.com/3742559/128940903-59a0b1cb-7a96-4b1a-a87c-1906dc63c07c.png)

##### Inkscape Preview

![inkscape preview](https://user-images.githubusercontent.com/3742559/128941114-2f9a9a6c-b236-4e44-8e9e-2f0fface3488.png)

Reply:
Thanks @ericcornelissen.. I will make these changes..

Reply:
Great improvement @akashpatil14298! This is almost good now, I'm just seeing some minor differences with the circle/dot in the middle of the logo. It looks like this is due to optimizations (as everything else lines up perfectly), could you retry this with the SVGO(MG) precision set one higher than before?

![image](https://user-images.githubusercontent.com/3742559/129271982-148b1027-16c7-427f-b51d-a3e24c01a1d9.png)